tests/test_performance.py

```diff
@@ -0,0 +1,239 @@
+from datetime import date, timedelta
+
+import numpy as np
+import pandas as pd
+import patsy
+import pymc3 as pm
+import pytest
+import theano.tensor as tt
+
+from pymc3_hmm.distributions import DiscreteMarkovChain, SwitchingProcess
+from pymc3_hmm.step_methods import FFBSStep
+from pymc3_hmm.utils import multilogit_inv
+
+
+def gen_toy_data(days=-7 * 40):
+    dates = date.today() + timedelta(days=days), date.today() + timedelta(days=2)
+    date_rng = pd.date_range(start=min(dates), end=max(dates), freq="H")
+    raw_data = {"dt": date_rng}
+    df = pd.DataFrame.from_dict(raw_data)
+    df = df.assign(
+        month=pd.Categorical(df.dt.dt.month, categories=list(range(1, 13))),
+        weekday=pd.Categorical(df.dt.dt.weekday, categories=list(range(7))),
+        hour=pd.Categorical(df.dt.dt.hour, categories=list(range(24))),
+    )
+    return df
+
+
+def create_poisson_zero_hmm(X, mu, xis, observed):
+    S = 2
+    z_tt = tt.stack([tt.dot(X, xis[..., s, :]) for s in range(S)], axis=1)
+    Gammas_tt = pm.Deterministic("Gamma", multilogit_inv(z_tt))
+    gamma_0_rv = pm.Dirichlet("gamma_0", np.ones((S,)))
+    V_rv = DiscreteMarkovChain("V_t", Gammas_tt, gamma_0_rv, shape=X.shape[0])
+    V_rv.tag.test_value = (observed > 0) * 1
+    Y_rv = SwitchingProcess(
+        "Y_t",
+        [pm.Constant.dist(0), pm.Poisson.dist(mu)],
+        V_rv,
+        observed=observed,
+    )
+    return Y_rv
+
+
+def create_NB_zero_hmm(X, mu, xis, alpha, observed):
+    S = 2
+    z_tt = tt.stack([tt.dot(X, xis[..., s, :]) for s in range(S)], axis=1)
+    Gammas_tt = pm.Deterministic("Gamma", multilogit_inv(z_tt))
+    gamma_0_rv = pm.Dirichlet("gamma_0", np.ones((S,)))
+    V_rv = DiscreteMarkovChain("V_t", Gammas_tt, gamma_0_rv, shape=X.shape[0])
+    V_rv.tag.test_value = (observed > 0) * 1
+    Y_rv = SwitchingProcess(
+        "Y_t",
+        [pm.Constant.dist(0), pm.NegativeBinomial.dist(mu, alpha)],
+        V_rv,
+        observed=observed,
+    )
+    return Y_rv
+
+
+def assert_performance(model, y_t):
```

Comment:
If we can use a more informative name for this helper function, that would help a lot.  As it's currently defined, this function asserts a correspondence between observations and a model's posterior predictive samples, so something along those lines would be better (e.g. `assert_posterior_predictives`).